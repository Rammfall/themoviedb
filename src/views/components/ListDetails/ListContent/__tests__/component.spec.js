import React from 'react'
import { shallow } from 'enzyme'

import ListContentComponent from '../component'

describe('ListContentComponent', () => {
  it('matches snapshot', () => {
    const props = {
      isEmpty: false,
      title: 'test',
      isEmptyTitle: 'test'
    }
    const component = shallow(
      <ListContentComponent
        {...props}
      />
    )

    expect(component).toMatchSnapshot()
  })
})
