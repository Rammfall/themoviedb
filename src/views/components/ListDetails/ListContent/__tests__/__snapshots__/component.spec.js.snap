// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ListContentComponent matches snapshot 1`] = `
<Fragment>
  <Row>
    <Col
      offset={2}
      span={20}
    >
      <div
        className="top-margin"
      >
        <Title>
          test
           
          <ForwardRef(MinusCircleOutlined) />
        </Title>
      </div>
    </Col>
  </Row>
  <RenderCtrl
    isEmpty={false}
    isLoading={false}
    renderEmpty={
      <EmptyState
        title="test"
      />
    }
    renderList={<withRouter(Connect(ListMovies)) />}
    renderLoading={<Loader />}
  />
</Fragment>
`;
